.github/workflows/release_windows.yml

```diff
@@ -63,7 +64,7 @@ jobs:
     - name: Setup QtIFW 4.x
       uses: jmarrec/setup-qtifw@v1
       with:
-        qtifw-version: '4.6.1'
+        qtifw-version: '4.8.1'
```

Comment:
there is NOT a windows arm64 runner yet available on github actions for free, but the QtIFW release 4.8.1 did add a windows arm64 installer, and I want consistency, so bump it up there too.

Reply:
Interesting.  Looks like it's in the April-June roadmap on GitHub Action's public roadmap.  (https://github.com/github/roadmap/issues/1098).  So we shouldn't be waiting long.